ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -398,6 +406,7 @@ PrjFS_Result PrjFS_WritePlaceholderFile(
     FILE* file = fopen(fullPath, "wx");
     if (nullptr == file)
     {
+        LogWarning("PrjFS_WritePlaceholderFile: fopen failed errorno=%d stderror=%s", errno, strerror(errno));
```

Comment:
Same comment here.  It looks like the `ENOENT` case is expected, could you move the `LogWarning` down to the `default` case below?